### DISCUSSION TITLE
v2.3.0

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#72] [Announcements]
- New Sources NetflixMirror and AutoEmbd
- Internal player improved
- All external Players support 
- Download folder shortcut
- Bugs fixed #70 

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.3.0'>v2.3.0</a>.</em>

**Comment by imsuchitrajena** [comment]
We Need More Hollywood Movie Providers Link.... It's Not That Update That We Looking Into Bug Fixes Are Okay But What We Want That's Not Yet Updated 🤔

**Reply by Zenda-Cross** [reply]
Most of the providers offer Hollywood movies, but if you are looking for English-only websites, you may have to wait for it. I can't guarantee it will be in the next update or not.
you also can try [Cloudstream ](https://cloudstream.miraheze.org/wiki/Main_Page)app it has many english sources.

**Comment by rohit82875** [comment]
Please add mkvcinemas

**Reply by Zenda-Cross** [reply]
Mkvcinemas has Cloudflare firewall so it's not possible to add it.

**Comment by Shebyyy** [comment]
Is it possible to add the ability to skip forward or backward by 10 seconds with a double tap on the right or left side of the screen?

**Reply by Zenda-Cross** [reply]
Yeah, I will try to add that if possible, but I'm not sure.

**Reply by Shebyyy** [reply]
Okay, thanks 🙏

**Comment by Oshsu2000** [comment]
Please make the app google tv/android tv compatible.
It'll be a breakthrough in smart tv era.

**Comment by LEOATIQ** [comment]
![20240813_153516.jpg](https://github.com/user-attachments/assets/7e54d8fc-c508-4261-a2d1-b38a2e8ef95e)

See this is what is looks smart tv

**Reply by Zenda-Cross** [reply]
that's probably because I forced its orientation to portrait.

**Reply by LEOATIQ** [reply]
Can  u fix it i like to watch in tv

**Comment by deep12323** [comment]
Please add something to bypass ISP

**Reply by Zenda-Cross** [reply]
I can't do anything to bypass ISP blocking
you have to use VPN or try changing dns

**Comment by LEOATIQ** [comment]
Sometimes big files movies  doesn't work at all like 75 gb movie that time it doesn't even stream

**Reply by Zenda-Cross** [reply]
I apologize, but I can't be of help with this server issue.

**Reply by LEOATIQ** [reply]
Okk

**Comment by rohit82875** [comment]
Please add scroll option for brightness and volume in Vega in built video player.

**Comment by rohit82875** [comment]
Please add scroll option for brightness and volume in Vega in built video player.

**Reply by Zenda-Cross** [reply]
Certainly, perhaps at a later time. For now, let's I am focusing on quality updates.

**Reply by LEOATIQ** [reply]
If I pause a video it will load the video can u add the option so i don't have to wait each sec to watch the video

**Comment by rohit82875** [comment]
Please also provide link of zip file to download all episodes of some show at once......and please add skymovieshd in your providers
